ember-app/app/mixins/cards/card-move.js

```diff
@@ -20,17 +20,19 @@ var CardMoveMixin = Ember.Mixin.create({
       var below_order = issue_below._data[this.get("orderKey")];
       return (above_order + below_order) / 2;
     },
-    moveToTop: function(issue_below){
-      var below_order = issue_below._data[this.get("orderKey")];
+    moveToTop: function(issue_below, key){
+      key = key ? key : this.get("orderKey");
```

Comment:
Actually.. not so straight-forward - I just remembered why its there in the first place . It's not impossible to acheive but will probably cause more changes than I would hope for in this PR.